packages/blocks/src/api/registration.js

```diff
@@ -25,12 +25,313 @@ import { isValidIcon, normalizeIconObject } from './utils';
+ *                                                 to identify a block. These
+ *                                                 can be any of the WordPress
+ *                                                 Dashicons, or a custom `svg`
+ *                                                 element.
+ * @property {string[]}              [keywords]    Sometimes a block could have
+ *                                                 aliases that help users
+ *                                                 discover it while searching.
+ *                                                 For example, an `image` block
+ *                                                 could also want to be
+ *                                                 discovered by `photo`. You
+ *                                                 can do so by providing an
+ *                                                 array of terms (which can be
+ *                                                 translated).
+ * @property {string[]}              [parent]      Setting `parent` lets a block
+ *                                                 require that it is only
+ *                                                 available when nested within
+ *                                                 the specified blocks.
+ * @property {FnSaveBlock}           save          Component describing
+ *                                                 serialized markup structure
+ *                                                 of a block type.
+ * @property {WPBlockStyle[]}        [styles]      Block styles can be used to
+ *                                                 provide alternative styles to
+ *                                                 block. It works by adding a
+ *                                                 class name to the block’s
+ *                                                 wrapper. Using CSS, a theme
+ *                                                 developer can target the
+ *                                                 class name for the style
+ *                                                 variation if it is selected.
+ * @property {WPBlockSupports}       [supports]    Optional block extended
+ *                                                 support features.
+ * @property {string}                title         This is the display title for
+ *                                                 your block, which can be
+ *                                                 translated with our
+ *                                                 translation functions. The
+ *                                                 block inserter will show this
+ *                                                 name.
+ * @property {WPBlockTransforms}     [transforms]  Block transformations.
+ */
+
+/**
+ * Editor block alignment options.
+ *
+ * @typedef {('left' | 'center' | 'right' | 'wide' | 'full')} WPBlockAlign
```

Comment:
Per consistency with below, and some previous conversations about minimizing line length, I assume we might want to remove the spaces around `|`.